tests/operators/derivative_operator.cpp

```diff
@@ -122,6 +122,47 @@ template <int D> void testDifferentiationABGV(double a, double b) {
     delete mra;
 }
 
+template <int D> void testDifferentiationCplxABGV(double a, double b) {
```

Comment:
doc :-)

Reply:
Explanation put it in the "FourComponents" PR